src/tools/CMakeLists.txt

```diff
@@ -126,5 +126,9 @@ if(WITH_SEASTAR)
   add_subdirectory(crimson)
 endif()
 
-add_subdirectory(immutable_object_cache)
-add_subdirectory(ceph-dencoder)
+if(NOT WIN32)
+  # Not available on Windows for now. We may enable it as soon as
+  # we switch from Unix sockes to Windows named pipes.
+  add_subdirectory(immutable_object_cache)
+  add_subdirectory(ceph-dencoder)
```

Comment:
Nit: this isn't tied to the immutable object cache -- it's a tester for the encode/decode of structs.